gradle/maven-to-gradle/src/main/java/com/sample/javacode/DisplayTime.java

```diff
@@ -0,0 +1,21 @@
+package com.sample.javacode;
+
+import java.util.Date;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class DisplayTime {
+
+	public static void main(String[] args) {
+		Date date = new java.util.Date();
+		System.out.println(date);
+		String gmt = date.toString();
+		boolean flag = StringUtils.isEmpty(gmt);
+		if(flag) {
+			System.out.println("No time found");
+		}else {
```

Comment:
please format code

Reply:
I have imported the eclipse formatter into my IDE and formatted the code. Please let me know if it's fine now.

Reply:
The format is correct now.